pkg/encoding/kzgEncoder/multiframe.go

```diff
@@ -0,0 +1,221 @@
+package kzgEncoder
+
+import (
+	"bytes"
+	"encoding/gob"
+	"errors"
+	"fmt"
+
+	rs "github.com/Layr-Labs/eigenda/pkg/encoding/encoder"
+	kzg "github.com/Layr-Labs/eigenda/pkg/kzg"
+	bls "github.com/Layr-Labs/eigenda/pkg/kzg/bn254"
+)
+
+// Sample is the basic unit for a verification
+// A blob may contain multiple Samples
+type Sample struct {
+	Commitment bls.G1Point
+	Proof      bls.G1Point
+	RowIndex   int // corresponds to a row in the verification matrix
+	Coeffs     []bls.Fr
+	X          uint // X is the same assignment index of chunk in EigenDa
```

Comment:
nit: `EigenDa` -> `EigenDA`

Also if it's chunk assignment index, can it be named as "ChunkIndex" for readability? "X" does not seem a proper name.

Reply:
I wil refactor the X a bit. The reason I avoid using assignment because, the pkg should relatively stand alone, X is the evaluation index

Reply:
refactored